aws-android-sdk-pinpoint/src/test/java/com/amazonaws/mobileconnectors/pinpoint/targeting/notification/GCMNotificationClientTest.java

```diff
@@ -152,14 +154,76 @@ private Bundle buildPushBundle() throws JSONException {
         return pushBundle;
     }
 
+    private Bundle buildJourneyPushBundle() throws JSONException {
```

Comment:
Can you break this up a bit with empty lines between the construction points?

Also, my faovrite way to bulid readable JSON is:
```
new JSONObject("key", new JSONObject()
    .put("innerKey", new JSONObject()
         .put("innerInnerkey", "innerInnerStringVal")
    )
).toString();
```
The indentation of the key names mirrors an actual JSON document and helps the reader understand the nesting levels with a quick glance.

Same suggestion applies to any place you are constructing JSON in this way.

Reply:
I like that. I went ahead and made the change and also cleaned up the other method that creates the payload for campaign tests